nagios/tests/common.py

```diff
@@ -3,8 +3,33 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
 import os
+from datadog_checks.dev import get_docker_hostname, get_here
 
-HERE = os.path.dirname(os.path.abspath(__file__))
+HERE = get_here()
+HOST = get_docker_hostname()
+
+CONTAINER_NAME = "dd-test-nagios"
+
+INSTANCE_INTEGRATION = {
+    'nagios_conf': '/etc/nagios4/nagios.cfg',
+    'collect_host_performance_data': True,
+    'collect_service_performance_data': False,
+}
+
+E2E_METADATA = {
+    'start_commands': [
+        'apt-get update',
+        'DEBIAN_FRONTEND=noninteractive apt-get install exim4-config',
+        'apt-get install nagios4-core -yq',
```

Comment:
Sorry I don't get it, you install nagios on the agent host? That's not the way it's supported to work.

Reply:
Nagios integration looks at multiple files on the host machine, the agent needs to be installed on the same machine as the service.